packages/ui/components/ui/select.tsx

```diff
@@ -13,20 +13,24 @@ const SelectValue = SelectPrimitive.Value;
 
 const SelectTrigger = React.forwardRef<
   React.ElementRef<typeof SelectPrimitive.Trigger>,
-  React.ComponentPropsWithoutRef<typeof SelectPrimitive.Trigger> & { open?: boolean }
->(({ className, children, open, ...props }, ref) => (
+  React.ComponentPropsWithoutRef<typeof SelectPrimitive.Trigger> & {
+    open?: boolean;
+    textColor?: string;
```

Comment:
The dropdown assumed always dark mode. Needed a way to specify its color.

Reply:
Hm, what do we think of adding a `colorMode` prop so that `SelectTrigger` defines its behavior in dark vs. light mode, rather than a parent defining its behavior?